packages/frontend/src/components/RichTextEditor/index.tsx

```diff
@@ -199,28 +199,30 @@ const RichTextEditor = ({
   const { control } = useFormContext()
   return (
     <FormControl style={{ flexGrow: 1 }}>
-      {label && (
-        <FormLabel isRequired={required} description={description}>
-          {label}
-        </FormLabel>
-      )}
-      <Controller
-        rules={{ required }}
-        name={name}
-        control={control}
-        defaultValue={defaultValue}
-        shouldUnregister={false}
-        render={({ field: { onChange, value } }) => (
-          <Editor
-            onChange={onChange}
-            initialValue={value}
-            editable={!disabled}
-            placeholder={placeholder}
-            variablesEnabled={variablesEnabled}
-            isRich={isRich}
-          />
+      <div data-test="text-input-group">
```

Comment:
nit: use `Box` instead of div?

Reply:
Cannot use `Box` or element from frameworks cos they don't expose `data-test` attrib

Reply:
oh wait i thought u did that for the `FlowStepHeader` in this [PR](https://github.com/opengovsg/plumber/pull/450/files)?

Reply:
Oh yea hor lemme double check whether I can do it right on the framework elem